### PR TITLE
SSO with JumpCloud

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
# What changed, and why it matters
Add instructions for JumpCloud SAML setup.

### CODE REVIEW

File: docs/platform/howto/saml/setup-saml-jumpcloud.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* Metadata URL
+* ACS URL
+
+Configure SAML on JumpCloud
+----------------------------
+
+#. In the `JumpCloud admin console <https://console.jumpcloud.com/login>`_, go to **SSO**.
+
+#. Select **Custom SAML App**.
+
```
Comment (chong-aiven) [inline docs/platform/howto/saml/setup-saml-jumpcloud.rst:28]: Set IDP id, e.g. https://jumpcloud.com/saml2/aiven

Comment (chong-aiven) [inline docs/platform/howto/saml/setup-saml-jumpcloud.rst:28]:
```suggestion
Line 28:  Set ``IdP Entity ID`` and ensure that the value you enter matches the Identity Provider Entity ID on Aiven's SSO configuration page. (example: https://jumpcloud.com/saml2/aiven)

Line 29: Set the ``Audience URI (SP Entity ID)`` with the ``Metadata URL`` from the Aiven Console. (suggest changing "to the" to "with the" 

Line 35: Add an entry in **Attribute statements** with ``Service Provider Attribute Name`` of ``email`` and ``JumpCloud Attribute Name`` of ``email``

Line 53:  Toggle ``IdP login`` button 

Line 56: Set the ``SAML IDP URL`` to the ``IDP URL`` field from JumpCloud.
Line 57:  Set the ``SAML Entity ID`` to the ``IdP Entity ID`` field from JumpCloud.
```

File: docs/platform/howto/saml/setup-saml-jumpcloud.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
```
Comment (chong-aiven) [inline docs/platform/howto/saml/setup-saml-jumpcloud.rst:28]: Set ``IdP Entity ID``. This is an unique case-sensitive identifier.

Comment (staceysalamon-aiven) [reply docs/platform/howto/saml/setup-saml-jumpcloud.rst:28]: How is this different from step 4 in this section: "Set the SAML Entity ID to the IdP Entity ID from JumpCloud."?

Comment (chong-aiven) [reply docs/platform/howto/saml/setup-saml-jumpcloud.rst:28]: Apologies for missing this. @staceysalamon-aiven The first step is "Set IdP Entity ID" is on Jumpcloud's console, while step 4 is on Aiven's console. 

Step 4 is about copying the URL from step 1 onto Aiven "SAML Entity ID" field

Comment (staceysalamon-aiven) [reply docs/platform/howto/saml/setup-saml-jumpcloud.rst:28]: @chong-aiven I'm certain I understood this correctly; I added a step to set the IdP entity ID after going to the customer SAML app in JumpCloud.

**Pull request merged**